x-pack/test/security_solution_endpoint/services/index.ts

```diff
@@ -7,9 +7,11 @@
 import { services as apiIntegrationServices } from '../../api_integration/services';
 import { services as xPackFunctionalServices } from '../../functional/services';
 import { EndpointPolicyTestResourcesProvider } from './endpoint_policy';
+import { IngestManagerProvider } from '../../common/services/ingest_manager';
 
 export const services = {
   ...xPackFunctionalServices,
   ingestManager: apiIntegrationServices.ingestManager,
   policyTestResources: EndpointPolicyTestResourcesProvider,
+  ingestManager: IngestManagerProvider,
```

Comment:
This does not look right - name collides with line 14

are you sure that `IngestManagerProvider` is not already in the `xPackFunctionalServices` that is spread above?